### PR TITLE
Add chain.json for Prysm

### PR DESCRIPTION
**Pull request by tsulhc**

### CODE REVIEW

File: prysm/chain.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    {
+      "kind": "PingPub",
+      "url": "https://explorer.kleomedes.network/prysm",
+      "tx_page": "https://explorer.kleomedes.network/prysm/tx/${txHash}",
+      "account_page": "https://explorer.kleomedes.network/prysm/account/${accountAddress}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/LightLABS-Dev/PRYSM-Branding/refs/heads/main/PRYSM%20LOGO%20PNG/PRYSM%20SPECTRUM%20NO%20BG.png",
```
Comment (JeremyParish69) [inline prysm/chain.json:151]: All referenced images will have to be uploaded to the Chain Registry repo

File: prysm/chain.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "prysm",
+  "status": "live",
+  "network_type": "testnet",
```
Comment (JeremyParish69) [inline prysm/chain.json:5]: Is this for a testnet? If so, the chain_name, directory name, and location will all need to change.

File: prysm/chain.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  "website": "https://www.prysm.network/",
+  "pretty_name": "Prysm",
+  "chain_type": "cosmos",
+  "chain_id": "prysm-devnet-1",
```
Comment (JeremyParish69) [inline prysm/chain.json:9]: Devnet is also an accepted network_type

**Comment by tsulhc** [comment]
@JeremyParish69 hope everything is solved by 4de0b4b009e76abe57d644a7be2801cc900491d2

File: testnets/prysmtestnet/chain.json [hunk 4]
```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "prysmtestnet",
```
Comment (JeremyParish69) [inline testnets/prysmtestnet/chain.json:3]: If this is a devnet as opposed to a testnet, then the chain_name and directory name should also be 'prismdevnet'.
(sorry for the confusing instruction, I wasn't sure which it was)

File: testnets/prysmtestnet/chain.json [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+      "version": "v0.50.8"
+    },
+    "ibc": {
+      "type": "go",
+      "version": "v8.4.0"
+    }
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/prysmtestnet/images/prysm.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/airchainstestnet/images/prysm.svg"
```
Comment (JeremyParish69) [inline testnets/prysmtestnet/chain.json:60]: it says 'airchains' here?

File: testnets/prysmtestnet/assetlist.json [hunk 6]
```diff
@@ -20,7 +20,7 @@
       "symbol": "PRYSM",
       "images": [
         {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/prysm/images/prysm.png",
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/prysmtestnet/images/prysm.png",
```
Comment (JeremyParish69) [inline testnets/prysmtestnet/assetlist.json:23]: why not include the svg here, too?

File: testnets/prysmtestnet/assetlist.json [hunk 7]
```diff
@@ -34,4 +34,4 @@
       "type_asset": "sdk.coin"
     }
   ]
```
[the remark this answers is not in the document]
Comment (tsulhc) [reply testnets/prysmtestnet/assetlist.json:36]: Thank you for your patience Jeremy!  In this case do you refer to line 3?

[the remark this answers is not in the document]
Comment (JeremyParish69) [reply testnets/prysmtestnet/assetlist.json:36]: yes

**Comment by tsulhc** [comment]
fingers crossed :D b80158ea06d6b8f28a2438530649ebfd76073067

File: testnets/prysmdevnet/chain.json [hunk 8]
```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "../chain.schema.json",
```
Comment (JeremyParish69) [inline testnets/prysmdevnet/chain.json:2]: relative location is "../../chain.schema.json"

File: testnets/prysmdevnet/assetlist.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "../assetlist.schema.json",
```
Comment (JeremyParish69) [inline testnets/prysmdevnet/assetlist.json:2]: relative location becomes "../../assetlist.schema.json"

**Pull request merged**